companion: return nextPagePath for drive/dropbox

stephentuso opened the pull request:
See #1611 - this is only the backend changes, the frontend will need an update for the issue to be completely resolved

Was the idea for all the `adapter.js` files to have the same interface? Drive/dropbox required different parameters, I updated the instagram `getNextPagePath` to match.

Team drive support appeared to be broken, so I made a few changes to get it working. Also, the current team drive API being used is deprecated and I think its usage can be simplified some, I didn't want too make to many changes re that in this PR though

arturi commented:
Hi, thanks for the PR! We’ll try to review as soon as possible.

ifedapoolarewaju commented on packages/@uppy/companion/src/server/provider/drive/index.js:
```diff
@@ -24,66 +24,80 @@ class Drive {
+
+    let where = {
+      fields: DRIVE_FILES_FIELDS,
+      pageToken: query.nextPageToken,
```
maybe we should make the nextPage query name the same for all providers? so each provider would use it as it requires internally. But between companion and uppy client it will be the same query name. For instagram we are already using `max_id` but we can change it and still support `max_id` for backward compatibility.

stephentuso commented:
@ifedapoolarewaju The team drive thing wasn't a typo but it was a bit redundant - I did some more testing and cleaned that up, the `teamDriveId` parameter isn't actually needed.

Also updated all providers to use `cursor` as the parameter

ifedapoolarewaju commented:
Thank you for the work! 💪 
created a task to implement the FE changes [here](https://app.asana.com/0/707684331679643/1127590983819333/f), but PRs are still welcome by all means 😬

stephentuso commented:
@ifedapoolarewaju No problem - has any progress been made on that task? If not I could get to it this week

ifedapoolarewaju commented:
@stephentuso no work has begun on this yet

ifedapoolarewaju commented:
@stephentuso FYI, I realized that the current frontend implementation should already work with the API changes you made in this PR, so there should be no need to add/change anything on the frontend
